packages/orch-skel/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@aglocal/orch-skel-contract",
+  "private": true,
+  "version": "0.1.0",
+  "description": "Smart contract component of orchestration skeleton",
+  "type": "module",
+  "files": [
+    "src",
+    "tools"
+  ],
+  "scripts": {
+    "build": "exit 0",
+    "test": "ava",
+    "test:c8": "c8 --all $C8_OPTIONS ava",
+    "test:xs": "exit 0",
+    "lint-fix": "yarn lint:eslint --fix",
+    "lint": "run-s --continue-on-error lint:*",
+    "lint:types": "tsc",
+    "lint:eslint": "eslint ."
+  },
+  "devDependencies": {
+    "@agoric/cosmic-proto": "^0.4.0",
+    "@agoric/ertp": "^0.16.2",
+    "@agoric/fast-usdc": "^0.1.0",
+    "@agoric/internal": "^0.3.2",
+    "@agoric/swingset-liveslots": "^0.10.2",
+    "@agoric/vow": "^0.1.0",
```

Comment:
@turadg when I use a new repository, I get a bunch of...

```
? Please choose a version of "@agoric/vow" from this list: 0.2.0-upgrade-20-dev-ef71
cfd.0
```

I guess I could use `dev` packages. But then it's another independent set of dependencies to track. hm.